src/app/sessions/containers/sessions-listings-search/sessions-listings-search.component.spec.ts

```diff
@@ -67,16 +69,33 @@ const mockedNotes: Note[] = [
         entityType: 'ListingRequest'
     }];
 
+const mockedHearingPartResponse: HearingPartResponse = {
+  id: 'some-id',
+  sessionId: undefined,
+  caseNumber: 'abc123',
+  caseTitle: 'some-case-title',
+  caseTypeCode: stubCaseType.code,
+  hearingTypeCode: stubHearingType.code,
+  duration: durationStringFormat,
+  scheduleStart: nowISOSting,
+  scheduleEnd: nowISOSting,
+  version: 2,
+  priority: Priority.Low,
+  reservedJudgeId: judgeId,
+  communicationFacilitator: 'interpreter',
+  deleted: false
+}
+
 const mockedUnlistedHearingPartVM: HearingPartViewModel = {
     id: 'some-id',
-    session: undefined,
+    sessionId: undefined,
     caseNumber: 'abc123',
     caseTitle: 'some-case-title',
     caseType: stubCaseType,
     hearingType: stubHearingType,
-    duration: moment.duration(sessionDuration),
-    scheduleStart: nowMoment,
-    scheduleEnd: nowMoment,
+    duration: moment.duration(durationStringFormat),
+    scheduleStart: moment(nowISOSting),
```

Comment:
Seems there's a typo - shouldn't it be String as in nowISOString?

Reply:
It is fine ;-)
duration is a time that tells how long it last, its also encoded is specific format eg  'PT30m'
scheduleStart and scheduleEnd makes a border when given hear part can occur

Reply:
I didn't mean changing the type. All I meant that there seems to be a typo - a spelling mistake ;)